### DISCUSSION TITLE
v0.28.2

### DISCUSSION THREAD

**Discussion by borg323** [LeelaChessZero/lc0#1669] [General]
This is what should have been v0.28.1:
* Improved cuda performance for 512 filter networks on Amprere GPUs.
* Several fixes for the onnx backend.
* New lc0 modes to process network files: describenet, leela2onnx and onnx2leela
* Documentation updates.
* Correctness fixes for rescorer support functions.

<hr /><em>This discussion was created from the release <a href='https://github.com/LeelaChessZero/lc0/releases/tag/v0.28.2'>v0.28.2</a>.</em>

**Comment by Arzam18** [comment]
Lco 0.28.2 for Android is Super Slow and on 8 cores have only more than 100 NPS in my Vivo y11😔. How to make it Faster at least more than 80,000 NPS?

**Comment by ednich** [comment]
I can't create an .uci file with Fritz 17 neither with cuda nor cudnn. Had no problems with opencl or dx12.

Is there a way to fix that? I own a Nvidia, so I would like to add them.

**Reply by mooskagh** [reply]
The usual way to debug the problem is to start `lc0.exe` without any chess GUI, type `go nodes 1` there and check whether there are any errors shown.

For further questions, I recommend to join our Dicsord chat at http://lc0.org/chat and ask there in the #help channel. That way it's usually much faster to find out what's wrong.

**Reply by ednich** [reply]
Thank you for the answer, I will do that.

Cheers.

edit: I got some errors starting the .exe without using the GUI (thank you for pointing that out): from cuda; cudart64_110.dll and cublas64_11.dll not found; and from cudnn: other library archives not found as well.

I will install CUDA Toolkit from Nvidia webpage, since I believe that's the way to solve it.

**Comment by MrPranks** [comment]
what's the difference between the nodll version and the version withuot the nodll termination?

**Reply by mooskagh** [reply]
It's the same, the nodll version just doesn't have cuda .dll files.
That's mainly for people with slow internet connection, they can download nodll version which is much smaller, and copy .dll files from the previous version manually.
